distributed/client.py

```diff
@@ -3961,6 +3961,39 @@ async def _get_task_stream(
         else:
             return msgs
 
+    async def _register_scheduler_plugin(self, plugin=None, idempotent=False, **kwargs):
+        if isinstance(plugin, type):
+            plugin = plugin(**kwargs)
+
+        return await self.scheduler.register_scheduler_plugin(
+            plugin=dumps(plugin, protocol=4)
+        )
+
+    def register_scheduler_plugin(self, plugin=None, idempotent=False, **kwargs):
+        """Register a scheduler plugin.
+
+        See https://distributed.readthedocs.io/en/latest/plugins.html
```

Comment:
Nitpick: we might instead link directly to the scheduler plugin section https://distributed.readthedocs.io/en/latest/plugins.html#scheduler-plugins